src/pages/home/report/ReportActionCompose.js

```diff
@@ -15,6 +21,8 @@ import {addAction, saveReportComment, broadcastUserIsTyping} from '../../../libs
 import ReportTypingIndicator from './ReportTypingIndicator';
 import AttachmentModal from '../../../components/AttachmentModal';
 
+const windowSize = Dimensions.get('window');
```

Comment:
Can you use the [`useWindowDimensions` hook](https://reactnative.dev/docs/usewindowdimensions) instead of `Dimensions.get('window')`? 

By doing something like this: 

```javascript
import { useWindowDimensions } from 'react-native';
const windowWidth = useWindowDimensions().width;
```

the value of `windowWidth` will automatically be updated when the window resizes. In your current solution, you measure the window width once before the component renders, so it doesn't get updated when you resize the window.

Reply:
`ReportActionCompose` is not a functional component.
How can use `useWindowDimensions` hook inside in a class component directly?

Reply:
You're right, my mistake